content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -11,12 +11,18 @@ This minimal install provides traffic management features of Istio.
 
 ## Prerequisites
 
-Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) described in the Quick Start guide.
+Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) described in the Quick Start guide.  The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
 
+    {{< tip >}}
+    To customize Istio and install addons, use the `--set <key>=<value>` option in the helm template or install command. [Installation Options](/docs/reference/config/installation-options/) references supported installation key and value pairs.
+    {{< /tip >}}
+    
 ## Installation steps
 
-You have two mutually exclusive options to install Istio:
-- To use Kubernetes manifests to deploy Istio, follow the instructions for [option #1](#option-1).
+The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
+
+Choose one of the following two **mutually exclusive** options described below.
+- To use Kubernetes manifests to deploy Istio, follow the instructions for [option #1](#option-1).		
```

Comment:
What is the key differentiation between the options? I could argue that option 2 also uses manifests, albeit through Tiller.
To help readers avoid confusion, be explicit about what one option needs that the other doesn't, in this case Tiller.
```suggestion
- To deploy Istio without using Tiller, follow the instructions for [option #1](#option-1).		
```